src/de/espend/idea/shopware/util/ShopwareUtil.java

```diff
@@ -53,47 +53,247 @@
         AND TABLE_NAME LIKE '%attributes'
      */
     final public static String[] MODEL_STATIC_ATTRIBUTES = new String[] {
```

Comment:
oh damn - my fault - i thought the missing tables in autocomplete are caused from the shopware plugin - but i think it only contains doctrine model tables :see_no_evil: